docs/live-queries.md

```diff
@@ -0,0 +1,1272 @@
+        id: user.id,
+        name: user.name,
+      })),
+  getKey: (user) => user.id, // Optional: uses stream key if not provided
+  startSync: true, // Optional: starts sync immediately
+}))
+```
+
+### Convenience Function
+
+For simpler cases, you can use `createLiveQueryCollection` as a shortcut:
+
+```ts
+import { createLiveQueryCollection, eq } from '@tanstack/db'
+
+const activeUsers = createLiveQueryCollection((q) =>
+  q
+    .from({ user: usersCollection })
+    .where(({ user }) => eq(user.active, true))
+    .select(({ user }) => ({
+      id: user.id,
+      name: user.name,
+    }))
+)
+```
+
+### Using with Frameworks
+
+In React, you can use the `useLiveQuery` hook:
+
+```tsx
+import { useLiveQuery } from '@tanstack/react-db'
+
+function UserList() {
+  const activeUsers = useLiveQuery((q) =>
+    q
+      .from({ user: usersCollection })
+      .where(({ user }) => eq(user.active, true))
+  )
+
+  return (
+    <ul>
+      {activeUsers.map(user => (
+        <li key={user.id}>{user.name}</li>
+      ))}
+    </ul>
+  )
+}
+```
+
+For more details on framework integration, see the [React](/docs/framework/react/adapter) and [Vue](/docs/framework/vue/adapter) adapter documentation.
+
+### Configuration Options
+
+| Option | Type | Description |
+|--------|------|-------------|
+| `id` | `string` | Unique identifier (auto-generated if not provided) |
+| `query` | `QueryBuilder` or function | The query definition |
+| `getKey` | `(item) => string \| number` | Function to extract keys from results |
+| `schema` | `Schema` | Optional schema for validation |
+| `startSync` | `boolean` | Whether to start syncing immediately |
+| `gcTime` | `number` | Garbage collection time in milliseconds |
+
+## Basic Queries
+
+The foundation of every query is the `from` method, which specifies the source collection or subquery. You can alias the source using object syntax.
+
+### Simple Queries
+
+Start with a basic query that selects all records from a collection:
+
+```ts
+const allUsers = createCollection(liveQueryCollectionOptions({
+  query: (q) => q.from({ user: usersCollection })
+}))
+```
+
+The result contains all users with their full schema. You can iterate over the results or access them by key:
+
+```ts
+// Get all users as an array
+const users = allUsers.toArray
+
+// Get a specific user by ID
+const user = allUsers.get(1)
+
+// Check if a user exists
+const hasUser = allUsers.has(1)
+```
+
+### Table Aliasing
+
+Use aliases to make your queries more readable, especially when working with multiple collections:
+
+```ts
+const users = createCollection(liveQueryCollectionOptions({
+  query: (q) => q.from({ u: usersCollection })
+}))
+
+// Access fields using the alias
+const userNames = createCollection(liveQueryCollectionOptions({
+  query: (q) =>
+    q
+      .from({ u: usersCollection })
+      .select(({ u }) => ({
+        name: u.name,
+        email: u.email,
+      }))
+}))
+```
+
+### Query Builder Pattern
```

Comment:
I'd expect this to show something new from the previous ones e.g. an `if (filter === 'foo') { /* add a where clause */ }`